release.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+#
+# Little script to release a new version.
+# Usage: release.sh x.y.z
+#
+# Dependencies: git, cargo-edit, ripgrep
+
+set -uo pipefail
+
+function update-cargo-versions()
+{
+    local version=$1
+
+    cargo set-version --workspace $version
+    git commit -am "chore: v$version"
+}
+
+function update-examples-versions()
+{
+    local version=$1
+
+    rg "shuttle-service = \{ version" --files-with-matches -g '!www/*' | xargs sed -i "s/shuttle-service = { version = \"[[:digit:]]*.[[:digit:]]*.[[:digit:]]*\"/shuttle-service = { version = \"$version\"/g"
```

Comment:
This strongly depends on the formatting of the `shuttle-service` entry in the `Cargo.toml` - would it be potentially less brittle to iterate over `examples/**/Cargo.toml` and run `cargo add shuttle-service@$version`?